superset-frontend/src/explore/ExplorePage.tsx

```diff
@@ -57,6 +61,43 @@ const fetchExploreData = async (exploreUrlParams: URLSearchParams) => {
   }
 };
 
+const getDashboardContextFormData = () => {
+  const dashboardPageId = getUrlParam(URL_PARAMS.dashboardPageId);
+  const sliceId = getUrlParam(URL_PARAMS.sliceId) || 0;
+  let dashboardContextWithFilters = {};
+  if (dashboardPageId) {
+    const {
+      labelColors,
+      sharedLabelColors,
+      colorScheme,
+      chartConfiguration,
+      nativeFilters,
+      filterBoxFilters,
+      dataMask,
+      dashboardId,
+    } =
```

Comment:
@kgabryje I believe this logic introduced a regression, i.e., `dataMask` could be `undefined` resulting in an error being thrown [here](https://github.com/apache/superset/blob/e3c6380258a46467ac8e09e64db60aad46bc4655/superset-frontend/src/dashboard/util/activeAllDashboardFilters.ts#L52). Would you be able to look into this issue?